akvo/rsr/tests/rest/test_permissions.py

```diff
@@ -510,6 +511,11 @@ def populate_model_map():
             'project_relation': 'period__indicator__result__project__'
         }
 
+        model_map[M.IndicatorPeriodDisaggregation] = {
+            'group_count': group_count(8, 2, 4, 4),
```

Comment:
I think this should read `(8, 2, 6, 4)` - I think we want org admins and project editors to be able to see entities on private projects. The permissions should be similar to `IndicatorPeriod` or `DisaggregationTarget`, I think. Please add relevant `rules.add_perm` entries to `rsr/models/__init__.py` to fix this.